pagopa/io-app
# feat: [#175272158] Refactored CIE PIN screen to functional, added informative bottom sheet

## List of changes proposed in this pull request
- Forgot PIN help is now a bottom sheet
- Refactored component to use accessible bottom sheet 

## How to test
- Login with CIE
- Test completing the procedure and canceling it
- See the bottom sheet by clicking "?" in the CIE PIN screen
